scripts/setup_sgtm_tasks_project.py

```diff
@@ -74,6 +74,15 @@ def resource_subtype(self):
         ],
     ),
     PeopleCustomField(name="Author (SGTM)"),
+    EnumCustomField(
+        name="Review Status",
+        enum_options=[
+            EnumOption(name="Needs Review", color="yellow"),
+            EnumOption(name="Changes Requested", color="red"),
+            EnumOption(name="Approved", color="green"),
+            EnumOption(name="Draft", color="blue")
```

Comment:
🤔 we already have a `Draft` enum for `PR Status`. Can this just be unpopulated if its a draft?
![image](https://github.com/Asana/SGTM/assets/22352112/0234ee5f-ab0d-49f5-b3cb-9c5cb006fa14)